filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
+    params = list(args)
+    params.append(a)
+    it = iter(params)
```

Comment:
please read my previous comments on this subject. It is an iterable, but not a listiterator (which is more or less a generator) and iter(list()) turns it into a listiterator. This is definitely needed.